experimental/Schemes/elliptic_surface.jl

```diff
@@ -1066,6 +1089,7 @@ function extended_ade(ADE::Symbol, n::Int)
 end
 
 function basis_representation(X::EllipticSurface, D::WeilDivisor)
+  error("not implemented")
```

Comment:
I think we should give this a try. It will throw an error if some intersection of divisors is not proper, but it should work for the generic case. And we could even try to work out the self-intersection thing.